.github/workflows/OSX.yml

```
@@ -152,12 +140,16 @@ jobs:
           make -j
           make install_sw
 
-      - name: Build
+      - name: Get OpenSSL path
         run: |
           export OPENSSL_ROOT_DIR=`pwd`/build/openssl/build
-          echo $OPENSSL_ROOT_DIR
-          make
-          rm build/release/src/libduckdb*
+          echo $OPENSSL_ROOT_DIR >> $GITHUB_ENV
```

Do we still need it? Maybe:

```suggestion
          echo "OPENSSL_ROOT_DIR=$OPENSSL_ROOT_DIR" >> $GITHUB_ENV
```

---

Yea good catch, added it!